src/settings/VoucherNumber/VoucherNumber.css

```diff
@@ -0,0 +1,6 @@
+@import "@folio/stripes-components/lib/variables.css";
```

Comment:
we don't have stripes-components in dependencies, are you sure this import will work?